loro-js/tests/tree.test.ts

```diff
@@ -1,12 +1,13 @@
 import { assert, describe, expect, it} from "vitest";
-import { LoroDoc, LoroTree, LoroTreeNode } from "../src";
+import { LoroDoc, LoroTree, LoroTreeNode, TreeDiff } from "../src";
 
 function assertEquals(a: any, b: any) {
   expect(a).toStrictEqual(b);
 }
 
 describe("loro tree", () => {
   const loro = new LoroDoc();
+  loro.setWithFractionalIndex(true);
```

Comment:
What's the default value for this?

Reply:
It's `false`, which means each fractional index will be `Default::default`.